app/models/user.rb

```diff
@@ -159,6 +159,22 @@ def can_moderate?
     admin? || moderator?
   end
 
+  # User status
+  # 0: banned
+  # 5: moderated
+  def mark_as!(new_status)
+    self.status = 0 if new_status == :banned
+    self.status = 5 if new_status == :moderated
```

Comment:
let's use ternary operator here for simplicity if this is only going to be about 2 states.

Reply:
Did you consider using an enum instead of adding this definition?
https://bendyworks.com/blog/rails-enum-sharp-knife

Reply:
In fact, 1 more state is missing. Check my first comment above (and please help find a good name :) ). 
Thank you for review.